charts/harvester-pcidevices-controller/Chart.yaml

```diff
@@ -15,13 +15,13 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.0-dev
+version: 0.2.1-rc1
```

Comment:
We can keep the original `version` and `appVersion` here and only update them in `release` branch backport.

Reply:
chart lint compares against previous commit and will fail. I can put the old versions back if you are happy with it.